src/components/InnerForm/InnerForm.tsx

```diff
@@ -35,7 +45,9 @@ const InnerForm: React.FC<InnerFormProps> = (props) => {
     }
 
     if (isHubspotDataForm(formData)) {
-        return <HubspotForm createDOMElement={true} {...formData.hubspot} />;
+        const contextProps = formsConfig as HubspotFormsContextProps;
+
+        return <HubspotForm createDOMElement={true} {...contextProps} {...formData.hubspot} />;
```

Comment:
and in this place `...formsConfig as HubspotFormsContextProps`

Reply:
done